base/views.py

```diff
@@ -1,15 +1,26 @@
 from django.shortcuts import render
 from django.http import HttpResponse
+from . models import Room
+from . forms import RoomForm
 
-rooms = [
-    {'id': 1, 'name': 'Lets learn python!'},
-    {'id': 2, 'name': 'Design with me'},
-    {'id': 3, 'name': 'Frontend developers'},
-]
+# rooms = [
```

Comment:
Now  you used static datas, but you should use dynamic datas using database.
like rooms  =  room.objects.all()